OptimizelySDK/Optimizely.cs

```diff
@@ -533,15 +542,15 @@ public string GetFeatureVariableString(string featureKey, string variableKey, st
         /// Sends impression event.
         /// </summary>
         /// <param name="experiment">The experiment</param>
-        /// <param name="variationId">The variation Id</param>
+        /// <param name="variation">The variation</param>
         /// <param name="userId">The user ID</param>
         /// <param name="userAttributes">The user's attributes</param>
-        private void SendImpressionEvent(Experiment experiment, string variationId, string userId, 
-        UserAttributes userAttributes)
+        private void SendImpressionEvent(Experiment experiment, Variation variation, string userId, 
```

Comment:
what's the reason to change from variationid to variation?

Reply:
Variation object is needed inside the function for passing it into the BroadcastExperimentActivated method.

We can get the object from Config.GetVariationFromId method but it adds an extra method call.  As the SendImpressionEvent method is private, I think there is no harm in modifying its signature unless the impact of this change is significant.

Reply:
ok